src/components/shared/Footer.tsx

```diff
@@ -1,39 +1,159 @@
+      <div className="row justify-content-center mt-5">
+        <div className="col-8">
+          <div className="row sponsers-pill">
+            <Sponsers/>
+          </div>
+          <WhoDisclaimer/>
+        </div>
+      </div>
+      <div className="row justify-content-center mt-5 pb-5 text-center">
+        <div>
+          <Link className="px-1" to={withLocaleUrl("PrivacyPolicy")}>{Translate('PrivacyPolicy')}</Link>
+        </div>
+        |
+        <div>
+          <Link className="px-1" to={withLocaleUrl("CookiePolicy")}>{Translate('CookiePolicy')}</Link>
+        </div>
+        |
+        <div>
+          <Link className="px-1" to={withLocaleUrl("TermsOfUse")}>{Translate('TermsOfUse')}</Link>
+        </div>
+      </div>
+    </footer>
+)
+
+const renderUpdateDate = (updatedAt: string) => {
+  return NumberDateToWordDate(updatedAt)
+}
+
+type UpdatedAtProps = {
+  updatedAt: string;
+}
+
+const UpdatedAt = ({updatedAt}: UpdatedAtProps) => {
+  // only renders 'last updated' when we have a valid date
+  return updatedAt ? (
+  <span>
+    {Translate("Footer", ["LastUpdated"])}: <b className='updated-at-bold'>{renderUpdateDate(updatedAt)}</b>
+  </span>
+  ): null
+}
+
+const Citation = () => (
+  <span className="text-right">
+    {Translate("Footer", ["CiteAs"])}
+    <a href={lancetId} target="__blank" rel="noopener noreferrer" className="cite-link">
+      <i>{Translate("Footer", ["LancetInfDis"])}</i> {Translate("Footer", ["Article"])}
+    </a> 
+  </span>
+)
+
+const WhoDisclaimer = () => (
+  <small className="whoDisclaimer">
+    {Translate('WhoSerotrackAndPartnersDisclaimerSmall')}
+  </small>
+)
+// to={withLocaleUrl(`${page}`)}
+const Sponsers = () => (
+  <div className="d-flex justify-content-around">
+    <a href="https://www.covid19immunitytaskforce.ca/" target="__blank" rel="noopener noreferrer" className="sponser-link">
+      <img src="https://www.covid19immunitytaskforce.ca/wp-content/themes/pena-lite-child/CITF_logo_ENG.svg" className="sponser-image" alt="COVID-19 Immunity Task Force Logo"></img>
+    </a>
+    <a href="https://cumming.ucalgary.ca/centres/centre-health-informatics" target="__blank" rel="noopener noreferrer" className="sponser-link">
+      <img src={UcalgaryLogo} className="sponser-image" alt="Centre for Health Informatics"></img>
+    </a>
+    <a href="https://www.canada.ca/en/public-health.html/" target="__blank" rel="noopener noreferrer" className="sponser-link">
+      <img src={HealthAgencyLogo} className="sponser-image" alt="Public Health Agency Logo"></img>
+    </a>
+    <a href="https://www.who.int/" target="__blank" rel="noopener noreferrer" className="sponser-link">
+      <WhoLogo className="sponser-image" />
+    </a>
+    <a href="https://joulecma.ca/" target="__blank" rel="noopener noreferrer" className="sponser-link">
+      <img src={AmcJoule} className="sponser-image" alt="CMA Joule"></img>
+    </a>
+  </div>
+)
+
+const PageLinks = () => (
+  <React.Fragment>
```

Comment:
Just curious: when do you use <React.Fragment> vs <></> (the empty tag) - Also if my understanding of Fragments is wrong here please correct me

Reply:
They're the same thing I beleive `<></>` is just shorthand